src/pages/index.js

```diff
@@ -112,51 +118,223 @@ const modalPreviewCloseButton = document.querySelector("#modalPreviewClose");
 //   document.removeEventListener("click", handleOverlayClose);
 // }
 
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  authToken: "23172f33-55e2-4e0e-a695-0bae3ab40106",
+});
+
+let cardSection;
+
+api.getInitialCards().then((cards) => {
+  cardSection = new Section({ items: cards }, renderCard, "#section");
+
+  function renderCard(cards) {
+    const cardElement = new Card(
+      cards,
+      "#card-template",
+      handleImageClick,
+      handleCardDeleteClick,
+      handleCardLike,
+      handleCardLikeRemove
+    );
+    console.log(cards);
+    cardSection.addItem(cardElement.getView());
```

Comment:
It's highly recommended to make a separate function `createCard` for creating a card not to copy-paste the code

```js		

function createCard(item) {
    // here you create a card
  return cardElement.getView()
}

```